### ISSUE TITLE
Adding click count functionality

### ISSUE THREAD

**Issue by anishka25** [TherkuTech/tturl#81]
I want to add click count functionality becuase it will help the users to track the no of people clicking their links.

**Comment by muruga21** [comment]
@anishka25 yeah it wil be a great feature.. but right now we are planning to add analytics to the urls.. i will confirm this feature and need to change schema too.. i will give some updates on this issue.. later...

so right now i cant assign this issue to you right now.. you can look into new bugs and features..

i hope you understand

thanks

**Comment by anishka25** [comment]
Can I work on analytics then? @muruga21
